src/miral/symbols.map

```diff
@@ -497,3 +497,10 @@ global:
     miral::ExternalClientLauncher::launch_using_x11*;
   };
 } MIRAL_2.8;
+
+MIRAL_2.10 {
+global:
+  extern "C++" {
+    miral::WindowSpecification::server_side_decorated*;
+  };
+} MIRAL_2.9;
```

Comment:
symbols.map should be updated via src/miral/regenerate-miral-symbols-map.py (or will be overwrritten).